<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>781</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="entrez">
    <attribute name="title">
     <string>Entrez Scraping</string>
    </attribute>
    <widget class="QRadioButton" name="proteinRadio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Protein</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="nucleotideRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nucleotide</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID List</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Entrez ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DNA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RNA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Amino Acid Sequence</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>410</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward Primer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>410</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reverse Primer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>410</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Assembly Forward Primer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>889</x>
       <y>410</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Assembly Reverse Primer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addtoList">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="exportLocationListScraping">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Location</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exportButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>390</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to .xlsx</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="assembliesCheck">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Design Assemblies</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="restrictionLine_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Restriction 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="restrictionLine_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Restriction 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idLineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>660</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QListWidget" name="idListWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="fwListWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="revListWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>430</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="dnaListWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="assemblyFwListWidget">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>430</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="rnaListWidget">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="aaListWidget">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>150</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="assemblyRevListWidget">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>430</y>
       <width>181</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Entrez Scraping Table</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QRadioButton" name="proteinRadio_3">
       <property name="text">
        <string>Protein</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="addtoTable">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="nucleotideRadio_3">
       <property name="text">
        <string>Nucleotide</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Add Entrez ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Export to .xlsx</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Plasmid Feature Mapping</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>821</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Plasmid ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Restriction Sites</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>231</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>530</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>852</x>
       <y>550</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Location</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>550</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLineEdit" name="emailSetting">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="defaultExport">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Default Export Location</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
